content/en/docs/reference/glossary/mirror-pod.md

```diff
@@ -9,13 +9,13 @@ aka:
 tags:
 - fundamental
 ---
- A {{< glossary_tooltip text="pod" term_id="pod" >}} object that a kubelet uses
+ A {{< glossary_tooltip text="pod" term_id="pod" >}} object that a {{< glossary_tooltip text="kubelet" term_id="kubelet" >}} uses
  to represent a {{< glossary_tooltip text="static pod" term_id="static-pod" >}}
 
 <!--more--> 
 
-When the kubelet finds a static pod in its configuration, it automatically tries to
+When the {{< glossary_tooltip text="kubelet" term_id="kubelet" >}} finds a static pod in its configuration, it automatically tries to
 create a Pod object on the Kubernetes API server for it. This means that the pod
 will be visible on the API server, but cannot be controlled from there.
 
-(For example, removing a mirror pod will not stop the kubelet daemon from running it).
+(For example, removing a mirror pod will not stop the {{< glossary_tooltip text="kubelet" term_id="kubelet" >}} daemon from running it).
```

Comment:
Better to leave these as they are. We only use a tooltip for the _first_ relevant use of a term.